assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,29 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+const myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
+const anotherFunction = (param) => {return param;}
```

Comment:
not incorrect, but you're not using all the advantages the fat arrow return offers. if you have only one parameter you can leave out the parentheses around it, you can leave out the curly braces around the actual function so long as it fits on one line, and with the implicit return you can actually leave that statement out. so this line would become 
```const anotherFunction = param => param```
which might look strange at first but will save you a  lot of time once you get the syntax down